devops/resources/run-ansible.sh

```diff
@@ -44,5 +44,17 @@ if [[ -n "${BECOME_USER}" ]]; then
   ANSIBLE_BECOME=" --become --become-user=${BECOME_USER} "
 fi
 
+# Test if docker shim flag file is present
+shim_enabled=true
 ansible ${ANSIBLE_PATTERN} ${ANSIBLE_INVENTORY} -u ${ANSIBLE_SSH_USER} ${ANSIBLE_BECOME} -m ${ANSIBLE_MODULE_NAME} \
--a "${ANSIBLE_MODULE_ARGS}"
+-a 'test -f /edx/etc/docker_shim_enabled' || echo "Docker shim is not enabled."; shim_enabled=false
```

Comment:
Is there a way to do this in a bash if condition or does it need to be passed in as module args to ansible?

Reply:
Oh - answered my question, it's on the box we are reaching out to, got it.

Reply:
Can you update the comment here to give an indication that it's the remote box being reached out to